p5/sketch/base.py

```diff
@@ -62,6 +56,10 @@
     config=config,
 )
 
+actual_width, actual_height = window.get_viewport_size()
```

Comment:
This seems to break the documentation auto-generation (see #50) I've pushed a fix on the ```fix-50-doc-autogen``` branch, can you see if it still works as expected on the Mac?